src/lib/forms-schema/index.ts

```diff
@@ -695,6 +695,7 @@ const formSchema = Joi.object().keys({
   postSubmissionReceipt: Joi.object({
     html: htmlString,
     allowPDFDownload: Joi.object(pdfSubmissionEventConfiguration),
+    allowAttachmentsDownload: endpointConfigurationSchema,
```

Comment:
I know the ship has sailed a bit, the the prop name doesn't match the value. Ie, allow sounds like it expects a boolean

Reply:
@Zaxist thoughts?